apcd-cms/src/apps/submission_form/templates/submission_form/submission_error.html

```diff
@@ -0,0 +1,69 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="container">
+    <div class="row">
+        <div class="col">
+            <h1>Request to Submit</h1>
+            <hr style="margin-top: 10px" />
+        </div>
+    </div>
+    <div class="c-message c-message--error">
+        There was an error when attempting to store your registration!
+        Please submit a ticket.
+    </div>
+    <form class="button-wrapper submit" action="/workbench/dashboard" method="GET">
+        <button class="form-button" type="submit">Go to Dashboard</button>
+    </form>
+</div>
+<style id="tup-175-css-alerts-messages-ui-pattern">
+    /*
+    Message
+    Inform user about a temporary status.
+    - .c-message--info - neither positive nor negative
+    - .c-message--success - a positive status
+    - .c-message--warning - a minor negative status
+    - .c-message--danger - a major negative status
+    Reference:
+    - https://getbootstrap.com/docs/4.0/components/alerts/
+    Styleguide Components.Message
+    */
+    /* @import url('../settings/color.css'); *//* ineffectual as live code, also settings are already available */
+    
+    /* Base */
+    
+    .c-message {
+      padding: 15px 20px;
+      border: var(--global-border--thick);
+      font-size: var(--global-font-size--small);
+    }
+    
+    /* Elements */
+    
+    .c-message a {
+      color: inherit;
+    }
+    
+    /* Modifiers */
+    
+    .c-message--info {
+      color: var(--global-color-info--dark);
+      background-color: var(--global-color-info--light);
+      border-color: var(--global-color-info--normal);
+    }
+    .c-message--success {
+      color: var(--global-color-success--dark);
+      background-color: var(--global-color-success--weak);
+      border-color: var(--global-color-success--normal);
+    }
+    .c-message--warning {
+      color: var(--global-color-warning--dark);
+      background-color: var(--global-color-warning--weak);
+      border-color: var(--global-color-warning--normal);
+    }
+    .c-message--error {
+      color: var(--global-color-danger--dark);
+      background-color: var(--global-color-danger--weak);
+      border-color: var(--global-color-danger--normal);
+    }
+    </style>
```

Comment:
**Code Organization.** Thank you for saving this to the repository.

Please move the `<style>` to `/apcd-cms/src/apps/tup-cms/src/taccsite_custom/tup-cms/static/templates/snippets/tup-175-css-alerts-messages-ui-pattern.html`.

<details><summary>Why?</summary>

1. This CSS should be in **one** place available to `submission_error.html` **and** `submission_form.html`.
2. This markup is already a snippet in the CMS on Prod. **and** QA servers (but should be saved to repo).
3. The snippet code can load from a template in a specific `templates/snippets` directory.
4. The specific `templates/snippets` directory will still save the code to repo.
5. I can easily find and remove the CSS in a snippet when I replace it with styles built into Core CMS (from Core-Styles).

</details>

Reply:
If I got that URL wrong, I take responsibility; but, regardless, the styling will still apply (via snippet) on QA and Prod. server and the code be saved to the repo.

You can test the repo-saved snippet code locally by adding a snippet to your local CMS, and setting the "TEMPLATE" field to be `snippets/tup-175-css-alerts-messages-ui-pattern.html`.